upgrading/upgrade-helm.adoc

```diff
@@ -7,7 +7,11 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [role="_abstract"]
-You can upgrade to the latest version of {product-title} from a supported older version. For upgrading to {product-title-short} 4.0, you must be using the latest patch release of {product-title-short} 3.74. If you are using an older version, you must first upgrade to {product-title-short} 3.74.
+When upgrading, follow this guidance:
```

Comment:
I'm nitpicking but this sentence looks very bare bones to me. How about if we make it a sub-section and call it like "upgrade sequence from ACS 3.x"?

Reply:
This is the abstract, which is a specific field that the Customer Portal requires to provide an overview of what is discussed in the module. I'll add something else for the abstract.